src/CalcManager/UnitConverter.cpp

```diff
@@ -65,8 +65,8 @@ UnitConverter::UnitConverter(_In_ const shared_ptr<IConverterDataLoader>& dataLo
     unquoteConversions[L"{sc}"] = L';';
     unquoteConversions[L"{lb}"] = LEFTESCAPECHAR;
     unquoteConversions[L"{rb}"] = RIGHTESCAPECHAR;
-    ClearValues();
-    ResetCategoriesAndRatios();
+    UnitConverter::ClearValues();
```

Comment:
I agree with Mingxin's comment. Couldn't see the motivation to get rid of polymorphism here.
Could you share more context?

Reply:
There is only one ClearValues function anyway, despite it being virtual. Polymorphism is not used for this class at all, so either we specify class or these functions should not be virtual

Reply:
Checking the code, `ClearValues()` and `ResetCategoriesAndRatios()` are not declared `virtual` at all. `ClearValues()` has nothing to do with polymorphism. `ResetCategoriesAndRatios` overrides the interface function but does not allow subclasses to override, therefore there's no polymorphism when calling `ResetCategoriesAndRatios()` from `UnitConverter` at runtime.